### PR TITLE
doc : add contribution guidelines

### LINKED ISSUE #22: Chore -> Add Contribution Guideline Docs

### PR DESCRIPTION
**Pull request by purohitdheeraj**
# Description

I have added contributing guidelines in CONTRIBUTING.md
some keys section added in this commit are
- how to contribute (fork, pull request)
- basic info on communication, issues, attribution

Fixes #22 

## Type of change

- [x] 📝 Documentation Update

## Added tests?
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   ```
+
+8. **Create a Pull Request**: Open a pull request from your fork to this main repository. Please provide a detailed description of your changes and reference any relevant issues if applicable.
+
+9. **Review and Discussion**: Your pull request will undergo review by the maintainers and other contributors. Be prepared to address feedback and make necessary changes.
+
+10. **Merge**: Once your pull request is approved, it will be merged into the main branch.
+
+## Coding Standards
+
```
Comment (ekas) [inline CONTRIBUTING.md:62]: Add more points here:

- Add Unit Test Cases
- Add text tokens for Internationalization (for this also add check box for PR template)
- Try to reuse existing components, if you don't find any existing please make one wherever necessary. 
- Please don't leave any Debugger & Console.log statements.
- Please comment your code for other developers where you feel logic is complex.
- Try to write code which is easily understandable by other fellow developers.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Please follow the coding standards and guidelines specific to the "JSLovers" project. If you're unsure about any coding standards, you can check the project's `CONTRIBUTING.md` or ask for clarification in your pull request or by reaching out to maintainers.
```
Comment (ekas) [inline CONTRIBUTING.md:63]:
```suggestion
Please follow the coding standards and guidelines specific to the "JSLovers" project. If you're unsure about any coding standards, or ask for clarification in your pull request or by reaching out to maintainers.
```

And this line to readme and create new section there with link to contributing guide.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Issues
+
+You can contribute not only through code but also by helping to identify and resolve issues. Feel free to browse our issue tracker, pick an issue that interests you, and leave a comment that you're working on it.
+
+## Communication
+
+If you have questions or need assistance, you can reach out to the project maintainers through GitHub issues or other communication channels, such as a dedicated Discord server or Slack workspace (if applicable).
```
Comment (ekas) [inline CONTRIBUTING.md:71]: Add the Discord link here or pass the readme section link for Discord here.

**Comment by purohitdheeraj** [comment]
Hey @ekas thanks for the feedback , based on that i have updated contributing guidelines and added a new section in `README` directing to `CONTRIBUTING`

**Review by ekas — APPROVED** [review]

**Pull request merged**